DevelopingSpace/starchart
# Add Worker/Queues to process notification based on status

## Description
This PR resolves #113, #115. 

### What's working?
- [x] Worker/Queue pair to run checks every 5 mins for DNS record and certs expiring in less than a month
- [x] Send Notification once.

### Steps to Test / Re-Test
- Delete  `redis` data and restart the Redis container (clear any previous jobs)
- `npm run setup` (to update to DB)
- Run the project (make sure Redis is up and running)
- `npm run db:studio` (to see the `lastNotified` field getting updated)
- Navigate to the MailHog interface at http://localhost:8025 to see notifications after 5 mins _(see Note)_

> **Note**
> To test if the jobs are repeating without waiting for 5 mins, modify the value of `repeat` [here](https://github.com/DevelopingSpace/starchart/pull/268/files#diff-db77765e8ad712b1a2c77389faf7bd213fc792da9cb30622f09719a57821e5a4R113) to every 5 seconds.
